templates.py

```diff
@@ -3531,6 +3531,90 @@
         },
     }),
 
+    ('german-noun-neuter-toponym', {
+        '@attribution': {'users': ['Lucas Werkmeister', 'Nikki'], 'title': 'Wikidata:Wikidata Lexeme Forms/German'},
+        'label': 'deutsches Substantiv (Neutrum, Toponym)',
+        'language_item_id': 'Q188',
+        'language_code': 'de',
+        'lexical_category_item_id': 'Q147276',
+        'forms': [
+            {
+                'label': 'Nominativ Singular',
+                'example': 'Das heutige [Berlin].',
+                'grammatical_features_item_ids': ['Q131105', 'Q110786'],
+            },
+            {
+                'label': 'Genitiv Singular',
+                'example': 'Fotos des damaligen [Berlins/Berlin].',
+                'grammatical_features_item_ids': ['Q146233', 'Q110786'],
+            },
+            {
+                'label': 'Dativ Singular',
+                'example': 'Fotos vom damaligen [Berlin].',
+                'grammatical_features_item_ids': ['Q145599', 'Q110786'],
+            },
+            {
+                'label': 'Akkusativ Singular',
+                'example': 'Plänen für das zukünftige [Berlin].',
```

Comment:
This should be „Pläne“, I believe.

```suggestion
                'example': 'Pläne für das zukünftige [Berlin].',
```